### PR TITLE
Some fixes (remove warnings, deprecations). Fix and improve README

### PR DESCRIPTION
**Pull request by wuarmin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,10 +32,9 @@ By default the input will be focused and have the content selected. If you prefe
 
 ## Running Tests
 
-    npm install
-    bower install
+    npm install (or yarn install)
     ember test
-    ember test --server
+    ember try:each (test again multiple ember versions defined in config/ember-try.js)
```
Comment (williamsbdev) [inline README.md]:
```suggestion
    ember try:each (test against multiple ember versions defined in config/ember-try.js)
```

**Comment by williamsbdev** [comment]
@wuarmin - This looks great! One very minor comment/suggestion to address and we'll get this merged and pushed.

**Comment by wuarmin** [comment]
@williamsbdev thanks. I applied your suggestion.

**Review by williamsbdev — APPROVED** [review]

**Comment by williamsbdev** [comment]
@wuarmin - Thanks for this PR! I've merged, tagged, and published version 2.2.0.

**Comment by wuarmin** [comment]
@williamsbdev thanks

**Pull request merged**